src/components/Checkbox/Checkbox.tsx

```diff
@@ -22,7 +22,6 @@ export interface CheckboxProps extends InputHTMLAttributes<HTMLInputElement> {
   readonly onChange?: React.ChangeEventHandler<HTMLInputElement>;
   readonly ref?: React.ForwardedRef<HTMLInputElement>;
   readonly id?: string;
-  readonly className?: string;
```

Comment:
You might want to check-in with @DorianKwan about removing class names as he just added className to all components in order to solve a Styled Components issue https://github.com/casper-network/casper-ui-kit/pull/173

Reply:
good catch! This was an error when merging in dev